go/host/storage/interfaces.go

```diff
@@ -44,6 +44,8 @@ type BatchResolver interface {
 	FetchBatchByHeight(height *big.Int) (*common.PublicBatch, error)
 	// FetchTotalTxCount returns the number of transactions in the DB
```

Comment:
maybe explain in the comment the purpose of these 2 methods.
A fast but inaccurate one to be used for tenscan, and the other for testing